hub/api/dataset.py

```diff
@@ -35,12 +34,6 @@ def __init__(
     ):
         """Initialize a new or existing dataset.
```

Comment:
Nitpick: the docstrings in the api should probably be changed to descriptive style as per the google docstrings guide.

`The docstring should be descriptive-style ("""Fetches rows from a Bigtable.""") rather than imperative-style ("""Fetch rows from a Bigtable."""), except for @property data descriptors, which should use the same style as attributes.`